pkg/shell/superuser.jsx

```diff
@@ -76,7 +76,11 @@ class UnlockDialog extends React.Component {
                 <Button variant='link' className='btn-cancel' onClick={state.cancel} isDisabled={!state.cancel}>
                     {state.message ? _("Close") : _("Cancel")}
                 </Button>
-                { state.busy && <div className="spinner pull-right" /> }
+                { state.busy &&
+                <span>
```

Comment:
You removed the pull-right class, so now the spinner does not go to the right, but rather after the buttons as it's in a flex container. It does not look bad, it's just inconsistent to what it was before I introduced the bug.
So you could either, add a dialog-wait-ct class and wrap it with a div exactly as the cockpit-components-dialog does, or pass the spinner as an idle_message to that dialog component, which will add this class for you. (I prefer the latter)

And btw, this is a bootstrap spinner so ideally we could port it to https://www.patternfly.org/v4/components/spinner
but unrelated.

Reply:
If you prefer to fix it in a follow up, the current position of the spinner works for me as well.